src/openrct2/interface/Theme.cpp

```diff
@@ -160,6 +160,7 @@ WindowThemeDesc WindowThemeDescriptors[] =
     { THEME_WC(WC_TITLE_OPTIONS),                  STR_THEMES_WINDOW_TITLE_MENU_OPTIONS,       COLOURS_3(TRANSLUCENT(COLOUR_DARK_GREEN),  TRANSLUCENT(COLOUR_DARK_GREEN),  TRANSLUCENT(COLOUR_DARK_GREEN)                     ) },
     { THEME_WC(WC_LAND_RIGHTS),                    STR_THEMES_WINDOW_LAND_RIGHTS,       COLOURS_3(COLOUR_DARK_YELLOW,              COLOUR_DARK_YELLOW,              COLOUR_DARK_YELLOW                                 ) },
     { THEME_WC(WC_THEMES),                         STR_THEMES_WINDOW_THEMES,       COLOURS_3(COLOUR_GREY,                     COLOUR_DARK_GREEN,               COLOUR_DARK_GREEN                                  ) },
+    { THEME_WC(WC_FOOTPATH_SCENERY),               STR_THEMES_WINDOW_FOOOTPATH_SCENERY,    COLOURS_3(COLOUR_GREY,                     COLOUR_LIGHT_BLUE,               COLOUR_LIGHT_BLUE                                  ) },
```

Comment:
Does this window _really_ need three colours? Seems like one would suffice. Also, make sure to test how it looks with the window colour set to translucent.